.github/workflows/api-specs-pr.yml

```diff
@@ -8,8 +8,13 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - name: Create Pull Request for API Specs
-        run: |
-          make api-specs-pr
+      - name: Set up Go
+        uses: actions/setup-go@v2
```

Comment:
A v3 is existing, should we use the updated version instead?

Reply:
I mean, we could, but we do use the `v2` somewhere else, I guess we could migrate everything in another PR.